docs/howto/usage.rst

```diff
@@ -134,6 +138,13 @@ Performance implications of the hipsolverSp API
   (:ref:`hipsolverSpXcsrlsvchol <sparse_csrlsvchol>` may perform slower and will require more memory usage than
   :ref:`hipsolverSpXcsrlsvcholHost <sparse_csrlsvcholHost>`.)
 
+- The functions :ref:`hipsolverSpScsrlsvqr <sparse_csrlsvqr>` are currently implemented by converting the sparse input matrix to a dense
+  matrix, and then running the dense factorization and linear solver on the result. This may result in slower-than-expected performance and
+  significant memory usage for large matrices.
```

Comment:
```suggestion
- The function :ref:`hipsolverSpScsrlsvqr <sparse_csrlsvqr>` is currently implemented by converting the sparse input matrix to a dense
  matrix, and then running the dense factorization and linear solver on the result. This might result in slower-than-expected performance and
  significant memory usage for large matrices.
```